state/pool_test.go

```diff
@@ -120,11 +111,102 @@ func (s *statePoolSuite) TestClose(c *gc.C) {
 
 	// Ensure that new ones are returned if further States are
 	// requested.
-	st1_, err := p.Get(s.ModelUUID1)
+	st1_, err := s.Pool.Get(s.ModelUUID1)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(st1_, gc.Not(gc.Equals), st1)
 
-	st2_, err := p.Get(s.ModelUUID2)
+	st2_, err := s.Pool.Get(s.ModelUUID2)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(st2_, gc.Not(gc.Equals), st2)
 }
+
+func (s *statePoolSuite) TestPutSystemState(c *gc.C) {
+	// Doesn't maintain a refcount for the system state.
```

Comment:
I guess I'm not - really this is testing that it doesn't blow up when someone Puts the system state. I'll add a check that it's not closed.